engine/swarm/admin_guide.md

```diff
@@ -53,6 +53,19 @@ updates to the swarm. A quorum of managers must also agree on node additions
 and removals. Membership operations are subject to the same constraints as state
 replication.
 
+### Maintaining a quorum of managers
+
+If the swarm loses a quorum, it will not be able to perform management
+tasks. If your swarm has multiple masters, always have more than two. In order
+to maintain a quorum, a majority of managers must be available. An odd number
+of managers is recommended, because the next even number does not make the quorum
+easier to keep. For instance, whether you have 3 or 4 managers, you can still only
+lose 1 manager and maintain a quorum. If you have 5 or 6 managers, you an still
+only lose two.
+
+See [Recovering from losing a quorum](recovering-from-losing-a-quorum) for
+troubleshooting steps if you do lose a quorum of managers.
```

Comment:
Same comment about "lose a quroum" / "losing a quorum", but again I'm not sure what's most correct.